packages/react/react/src/components/OpenGraph/index.spec.tsx

```diff
@@ -0,0 +1,30 @@
+import { render } from '@testing-library/react';
+import { OpenGraph } from '.';
+
+describe('OpenGraph', () => {
+  const getMeta = (metaName: string) => {
+    const metas = document.getElementsByTagName('meta');
+    for (let i = 0; i < metas.length; i += 1) {
+      if (metas[i].getAttribute('property') === metaName) {
+        return metas[i].getAttribute('content');
+      }
+    }
+    throw new Error('cannot find your meta tag');
+  };
+  it(`should render inserted title`, () => {
+    render(<OpenGraph title="토스" />);
+    expect(getMeta('og:title')).toEqual('토스');
```

Comment:
```suggestion
    const title = '토스'

    render(<OpenGraph title={title} />);
    expect(getMeta('og:title')).toEqual(title);
```

Reply:
Your suggestion is so clear
it cannot make human mistake
I will apply your suggestion 
Thank you!